Differential/willow-prototype
# Update upcoming events screen

## DESCRIPTION
This PR improves the upcoming events screen. Screenshot below represents the changes.

<img src="https://user-images.githubusercontent.com/2528817/70836105-ca1e4b80-1dc4-11ea-8f42-5f9c98b1e7f5.png" height="600">


### Screen captures of screen
iOS
https://v.usetapes.com/qEiAHMjUrm
Android
https://v.usetapes.com/mLtDEd658P

### What design trade-offs/decisions were made?
1. Font weight and colors are different from the Figma design to match the existing styles of the upcoming events on the main app screen. This should be flagged as an item to sync design and core

### How do I test this PR?
1. Go to main screen in app scroll down to "Here's what's coming up!" section
2. Click `View More`
3. See improved Upcoming Events screen.

## TODO:


## REVIEW:

**Manual QA**


**Code Review: Questions to consider**


> The purpose of PR Review is to _improve the quality of the software._
